py-test/request.py

```diff
@@ -80,7 +80,8 @@ def test_post_put_delete_course(self):
                           headers={'content-type': 'application/json', 'Authorization': self.admin_auth_header})
         
         self.assertEqual(type(r.json()), type({}))
-        self.assertEqual(1, len(r.json()))
+        pprint(r.json())
```

Comment:
Is the output necessary?

Reply:
Mh, yes, I can delete this for travis, also all the other prints...